frontend/src/routes/index.js

```diff
@@ -0,0 +1,36 @@
+import { HashRouter, Routes, Route } from "react-router-dom";
+import Footer from "../components/Footer";
+
+import Header from "../components/Header";
+import Homepage from "../pages/Homepage";
+import Login from "../pages/Login";
+import Register from "../pages/Register";
+import PageNotFound from "../pages/PageNotFound";
+
+const Directory = () => {
```

Comment:
since this is an `index.js` file, please put same name as `Routes` instead of directory

Reply:
it would be hard since Routes is already a used name in react-router